config/initializers/rollbar.rb

```diff
@@ -0,0 +1,53 @@
+Rollbar.configure do |config|
+  config.access_token = Rails.application.secrets.rollbar_token
+
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+  config.enabled = Rails.env.production? && config.access_token.present?
+
+  # By default, Rollbar will try to call the `current_user` controller method
+  # to fetch the logged-in user object, and then call that object's `id`,
+  # `username`, and `email` methods to fetch those properties. To customize:
+  config.person_method = nil
```

Comment:
Couldn't find any documentation on how to disable this. 

But seems like they do an if check on this method and then proceed with setting up the personal data, so setting this to nil should prevent this:
https://github.com/rollbar/rollbar-gem/blob/d3ca4fbeceee7621cbe8964ad1e160d247c84afe/lib/rollbar/plugins/rails/controller_methods.rb#L9